src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+import mate.academy.model.User;
+
+public class AuthenticationService extends UserService {
```

Comment:
why does it extend UserService? we don't need it)

Reply:
was thinking that it needs to be connected to each other? but will fix it now